docs/index.html

```diff
@@ -95,6 +95,25 @@
             min-width: 120px;
         }
 
+        .year_button {
+            background-color:#0d47a1;
+            display:block !important;
+            width: 50%;
```

Comment:
Style wise for a PC, could we just make the minimizer a small icon like [this](https://feathericons.com/?query=minimize). Its a style choice though.

Reply:
I dont...understand. I thought we wanted a button that shows the year?